index.js

```diff
@@ -1,18 +1,69 @@
 /***** Deliverable 1 *****/
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
-
+// Need to include defer in my html doc, script tag, to differ the execution of 
+// the JS file until it finishes loading the HTML body file.
 
 /***** Deliverable 2 *****/
-
+// to find the write attribute I console.dir(header) and find the style.color attribute.
+ header.style.color = 'blue'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+console.log('This is what the traveler object looks like: ', traveler)
+const travelerImg = document.querySelector('div#profile img')
+travelerImg.src = traveler.photo
+travelerImg.alt = traveler.nickname
+const travelerName = document.querySelector('div#profile h2')
+travelerName.innerText = traveler.name
```

Comment:
You can also use `textContent` instead of `innerText`. The main difference is that `textContent` doesn't consider the CSS whereas `innerText` does. [Take a look at this stackoverflow post about it](https://stackoverflow.com/questions/35213147/difference-between-textcontent-vs-innertext)